smac/utils/multi_objective.py

```diff
@@ -0,0 +1,57 @@
+from typing import Union, List, Tuple, Optional
+import numpy as np
+
+
+def normalize_costs(
+    values: Union[np.ndarray, List, List[List]],
+    bounds: Optional[List[Tuple[float, float]]] = None,
+) -> np.ndarray:
+    """Normalizes the costs to be between 0 and 1 if no bounds are given.
+    Otherwise, the costs are normalized according to the bounds.
+
+    Parameters
+    ----------
+    values : Union[np.ndarray, List, List[List]]
+        Cost values which should be normalized.
+        If array/list is one-dimensional, it is expanded by one dimension.
+    bounds : Optional[List[Tuple[float, float]]], optional
+        Min and max bounds which should be applied to the values, by default None.
+        If bounds are None the min and max values from the data are used.
+
+    Returns
+    -------
+    np.ndarray
+        Normalized costs
+    """
+
+    if isinstance(values, list):
+        values = np.array(values)
+
+    if len(values.shape) == 1:
+        values = np.expand_dims(values, axis=0)
```

Comment:
Why does `costs = [3,4,5]` need to be expanded? Is this treated as the cost of a single run with three multi-objective costs `[3,4,5]` or as three separate runs with each having a cost `3`, `4` and `5`?

Reply:
[3,4,5] means three runs. Therefore, we want to have [[3], [4], [5]]. Added examples in the docstring.

Reply:
But this line result in [[3, 4, 5]]